fec.config.js

```diff
@@ -19,14 +18,6 @@ module.exports = {
    // NOTE: This is here for local testing purposes
   // remove it when you want to use deployed chrome-service,
   // meaning fed-modules and navigation is updated in chrome-service-backend
-  routes: {
-    '/api/chrome-service/v1/static': {
-      host: 'http://localhost:8090',
-    },
-    '/api/roadmap/v1': {
-      host: 'http://localhost:8000/',
-    },
-  },
```

Comment:
Removing this (and the line 6) actually help and the frontend runs locally fine. We just need to remove only the `chrome-service` route and keep the API route -> we want to use locally provided backend.

I already experimented with this a bit, but there was an outage of stage for providing the chrome-sevice data (it was super slow and it most cases the locally running page didn't load at all) when using BRQ2 VPN - it seemed this change is not working, but it was different issue.

Local frontend running with this change of `fec.config`:
<img width="1800" height="908" alt="Screenshot From 2025-08-03 21-50-39" src="https://github.com/user-attachments/assets/b2576f11-1fab-43d4-86e6-0be4affead61" />